### PR TITLE
ARROW-7034: [CI][Crossbow] Skip known nightly failures

### PR DESCRIPTION
**Pull request by nealrichardson**

### CODE REVIEW

File: dev/tasks/tasks.yml [hunk 1]
```diff
@@ -195,7 +195,7 @@ groups:
     - docker-docs
     - docker-lint
     - docker-iwyu
-    - docker-clang-format
+    # - docker-clang-format # ARROW-6914
```
Comment (kou) [inline dev/tasks/tasks.yml]: It seems that 583fb7e5cd7b8d504cc0645949e4d367ff55b607 merges `clang-format` into `lint`. We may be able to just remove this entry:

```diff
diff --git a/dev/tasks/tasks.yml b/dev/tasks/tasks.yml
index 5138853d0..d35568ae2 100644
--- a/dev/tasks/tasks.yml
+++ b/dev/tasks/tasks.yml
@@ -102,7 +102,6 @@ groups:
     - docker-docs
     - docker-lint
     - docker-iwyu
-    - docker-clang-format
     - docker-pandas-master
     - docker-dask-integration
     - docker-hdfs-integration
@@ -1590,22 +1589,6 @@ tasks:
         - docker-compose build lint
         - docker-compose run iwyu
 
-  docker-clang-format:
-    ci: circle
-    platform: linux
-    template: docker-tests/circle.linux.yml
-    params:
-      environment:
-        PYTHON_VERSION: 3.6
-      commands:
-        - docker-compose pull --ignore-pull-failures cpp
-        - docker-compose pull --ignore-pull-failures python
-        - docker-compose pull --ignore-pull-failures lint
-        - docker-compose build cpp
-        - docker-compose build python
-        - docker-compose build lint
-        - docker-compose run clang-format
-
   ############################## Integration tests ############################
 
   docker-dask-integration:
```

@fsaintjacques What do you think about this?

(We may be able to remove `docker-iwyu` too.)

Comment (fsaintjacques) [reply dev/tasks/tasks.yml]: Yes we can remove this task. iwyu was not merged into the lint image; I wouldn't remove it yet.

**Comment by nealrichardson** [comment]
Removed, PTAL @fsaintjacques @kou

**Review by kou — APPROVED** [review]
+1

**Pull request closed without merging**